ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -257,38 +258,98 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  getRemainingTime(firstDate, secondDate) {
```

Comment:
```suggestion
  getTimeDifference(firstDate, secondDate) {
```

Reply:
I left a similar comment. This was marked as resolved because of the recent changes.